test/NuGet.Core.FuncTests/NuGet.Signing.CrossFramework.Test/CrossVerifyTestFixture.cs

```diff
@@ -19,7 +19,7 @@ public class CrossVerifyTestFixture : IDisposable
         //In net472 code path, the SDK version and TFM could not be detected automatically, so we manually specified according to the sdk version we're testing against.
         //https://github.com/NuGet/Home/issues/12187
         private const string SdkVersion = "7";
-        private const string SdkTfm = "net5.0";
+        private const string SdkTfm = "net7.0";
```

Comment:
I'm curious:  is this change directly or coincidentally related to the PR?  It wasn't in earlier iterations.

Reply:
It is directly related. Cross Verify tests were failing without this change. 
Due to how these tests are setup, the tricks for detecting the framework in the other integration tests were not appropriate.